test/scrabble_test.rb

```diff
@@ -6,8 +6,28 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    assert_equal 1, Scrabble.new.score("A")
+    assert_equal 4, Scrabble.new.score("F")
+  end
+
+  def test_it_can_score_a_word
+    assert_equal 6, Scrabble.new.score('BUTT')
+    assert_equal 32, Scrabble.new.score('QUETZALCOATL')
+  end
+
+  def test_it_can_score_regardless_of_case
+    assert_equal 6, Scrabble.new.score('ButT')
+    assert_equal 32, Scrabble.new.score('QuEtZAlCOAtL')
+  end
+
+  def test_knows_if_valid_word
```

Comment:
Love that you're testing at the unit level. If you break out `#calculate_score` as I described above, should be able to unit test that as well.